templates/opensshd.conf.j2

```diff
@@ -107,14 +107,13 @@ HostbasedAuthentication no
 {% if ssh_pam_support -%}
 UsePAM {{ 'yes' if (ssh_use_pam|bool) else 'no' }}
 {% endif %}
-{% if ssh_google_auth %}
-# Force public key auth then ask for google auth code
-AuthenticationMethods publickey,keyboard-interactive
-{% endif %}
 
-# Force public key auth then ask for pam device input
-{% if ssh_pam_device %}
-AuthenticationMethods publickey,keyboard-interactive:pam
+# Set AuthenticationMethods per default to publickey
+AuthenticationMethods {{ sshd_authenticationmethods|default("publickey") }}
```

Comment:
`|default("publickey")` is unnecessary here since its set by default in `defaults`.

Reply:
Fixed